lib/Test2/Harness/Collector.pm

```diff
@@ -271,6 +308,9 @@ sub jobs {
         );
     }
 
+    # The collector didn't read in all the jobs because it'd run out of file handles. We need to let the stream know we're behind.
+    $self->send_backed_up if $max_open_jobs <= keys %$jobs;
```

Comment:
This will happen when backed up. It should be a different message than above. Ideally a small message to know that things are backed up here should be displayed every time it's backed up.

Reply:
Take a look at my multiple-collector branch which has a better way to handle this. That branch is unstable, but I am going to bring some of the logic for determining when the issue happens back into this branch tonight when I have a chance. It should be better than what is here.